src/main/kotlin/no/nav/familie/ef/mottak/api/EttersendingController.kt

```diff
@@ -22,18 +22,21 @@ import org.springframework.web.bind.annotation.RestController
 import org.springframework.web.multipart.MultipartFile
 
 @RestController
-@RequestMapping(consumes = [MULTIPART_FORM_DATA_VALUE, APPLICATION_JSON_VALUE],
-                path = ["/api/ettersending"],
-                produces = [APPLICATION_JSON_VALUE])
+@RequestMapping(path = ["/api/ettersending"], consumes = [APPLICATION_JSON_VALUE], produces = [APPLICATION_JSON_VALUE])
 @Protected
 class EttersendingController(val ettersendingService: EttersendingService) {
 
     private val logger = LoggerFactory.getLogger(this::class.java)
     private val secureLogger = LoggerFactory.getLogger("secureLogger")
 
     @PostMapping
-    fun ettersend(@RequestPart("ettersending") ettersending: EttersendingMedVedlegg,
-                  @RequestPart("vedlegg", required = false) vedleggListe: List<MultipartFile>?): Kvittering {
+    fun ettersend(@RequestPart("ettersending") ettersending: EttersendingMedVedlegg): Kvittering {
+        return okEllerKastException { ettersendingService.mottaEttersending(ettersending) }
+    }
+
+    @PostMapping(consumes = [MULTIPART_FORM_DATA_VALUE])
+    fun ettersendOld(@RequestPart("ettersending") ettersending: EttersendingMedVedlegg,
+                     @RequestPart("vedlegg", required = false) vedleggListe: List<MultipartFile>?): Kvittering {
```

Comment:
Tenker nesten vi kan fjerne denne (og deprecated funksjonalitet i ´EttersendingService´), siden vi ikke er i prod enda. Hva synes du?

Reply:
Sant, tenker jeg kan slette det sen når jeg sletter "de samme" for søknaden :)